打包失败

```
  Restore completed in 1.85 sec for D:\a\dotnetCampus.WeChatWork\dotnetCampus.WeChatWork\src\dotnetCampus.WeChatWork\dotnetCampus.WeChatWork.csproj.
##[error]C:\Program Files\dotnet\sdk\3.1.201\Sdks\NuGet.Build.Tasks.Pack\build\NuGet.Build.Tasks.Pack.targets(198,5): error NU5026: The file 'D:\a\dotnetCampus.WeChatWork\dotnetCampus.WeChatWork\src\dotnetCampus.WeChatWork\bin\Release\netcoreapp3.1\dotnetCampus.WeChatWork.dll' to be packed was not found on disk. [D:\a\dotnetCampus.WeChatWork\dotnetCampus.WeChatWork\src\dotnetCampus.WeChatWork\dotnetCampus.WeChatWork.csproj]
##[error]Process completed with exit code 1.
```

https://github.com/dotnet-campus/dotnetCampus.WeChatWork/runs/607537083?check_suite_focus=true

--- a/.github/workflows/nuget-publish.yml
+++ b/.github/workflows/nuget-publish.yml
@@ -14,7 +14,7 @@ jobs:
       - name: Setup
         uses: actions/setup-dotnet@v1
       - name: Pack
-        run: dotnet pack --configuration Release
+        run: dotnet build --configuration Release
       - name: Setup Nuget.exe
         uses: warrenbuckley/Setup-Nuget@v1
       - name: Add private GitHub registry to NuGet

